docs/WebView/README.en.md

```diff
@@ -122,6 +122,27 @@ When performing an iOS build in certain Unity versions, the buttons on **Navigat
 
 ![GPMWebViewBundle.png](images/GPMWebViewBundle.png)
 
+#### Optional automation
+
+You can automate above process by using [OnPostprocessBuild](https://docs.unity3d.com/ScriptReference/Build.IPostprocessBuildWithReport.OnPostprocessBuild.html) and [PBXProject](https://docs.unity3d.com/ScriptReference/iOS.Xcode.PBXProject.html).
```

Comment:
Since the contents can depend on the Unity version, why not specify the verified Unity version?

Reply:
That's a good idea. I'm using Unity 2020.3.21f1, without iOS setup, the navigation buttons are not visible.

I haven't tested on other Unity versions yet, so for now, can I specify this version?

Reply:
Any way would be great as long as users can check the version in the guide
For example :
Note - Checked Unity Version 2020.3.21f1

Reply:
OK!!

Reply:
It's added! 👍 

Thank you.